apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetDAO.java

```diff
@@ -84,4 +86,12 @@ List<Dataset> find(@Bind("limit") int limit,
     int[] recordExperiments(@Bind("workspace_id") String workspaceId,
             @BindMethods Collection<DatasetLastExperimentCreated> datasets);
 
+    @SqlQuery("SELECT workspace_id, created_by AS user, COUNT(DISTINCT id) AS count " +
+            "FROM datasets " +
+            "WHERE DATE(created_at) = DATE(NOW() - INTERVAL 1 DAY) " +
+            "GROUP BY workspace_id,created_by")
+    List<BiInformationResponse.BiInformation> getExperimentBIInformation();
```

Comment:
This query will result in a full table scan. FYI @andrescrz, Could we retrieve this by workspace at least? Otherwise, we may need to add an index.

Reply:
We need to get daily data for all workspaces and users. We could try to optimize it by caching users/workspaces daily and use them in query, but I guess it's an overhead at this moment.

I tested `traces` in production and this query runs almost instantly and results in only 84 rows, so it doesn't consume a lot of memory or processing time. Other two tables are smaller. Keep in mind that it returns only new entries for the previous day, not all data. 

Also this will be executed only once per day by a cron job and execution time could be bigger, as long as it doesn't cause HTTP request timeout. And we are really far from that point now. If we ever approach it we could consider implementing BI events sending directly from Opik, instead of EM. As for now this is an easy workaround.

And we definitely don't want to add an index for `created_at` field for this, it will slow down data insertions.